scripts/setup-obproxy.sh

```diff
@@ -0,0 +1,251 @@
+#! /usr/bin/env bash
+
+# This script is used to setup obproxy for an OBCluster handyly for quick testing.
+# It will install obproxy and configure it with a simple configuration file.
+
+VERSION=0.1.0
+NAMESPACE=default
+DEPLOY_NAME=""
+PROXY_VERSION="4.2.1.0-11"
+DESTROY=false
+NODE_PORT=false
+DISPLAY_INFO=false
+CONNECT=false
+
+function print_help {
+  echo "setup-obproxy.sh - Set up obproxy for an OBCluster in a Kubernetes cluster"
+  echo "Usage: setup-obproxy.sh [options] <OBCluster>"
+  echo "Options:"
+  echo "  -h, --help            Display this help message and exit."
+  echo "  -v, --version         Display version information and exit."
+  echo "  -n                    Namespace of the OBCluster. Default is default."
+  echo "  -i, --info            Display the obproxy deployment information."
+  echo "  -d, --deploy-name     Name of the obproxy deployment. Default is obproxy-<OBCluster>."
+  echo "  --connect             Connect to the obproxy deployment."
```

Comment:
There is a connect.sh, add a parameter in connect.sh and pass the deploy name to that script